Ginger/GingerCoreCommon/ALM/ALMIntegration.cs

```diff
@@ -29,7 +29,8 @@ public enum eALMType
             RALLY = 3,
             Jira = 4,
             Qtest = 5,
-            Octane = 6
+            Octane = 6,
+            ZephyrEnt = 7
```

Comment:
Please use full name ZephyrEnterprise instead of Abbreviation

Reply:
done